### ISSUE TITLE
Extension Preview

### ISSUE THREAD

**Issue by sofari23** [null2264/yokai#22]
### Describe your suggested feature

Would it be possible to enter WebView without installing the extension as a sort of preview like in mihon?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an request for a feature in the original Tachiyomi, I am opening because multiple versions of Yōkai have been released without this feature.
- [X] If this is an issue with an extension, or a request for an extension, I should be contacting the extensions repository's maintainer/support for help.
- [X] I have updated the app to version **[1.7.12](https://github.com/null2264/yokai/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

**Comment by null2264** [comment]
Not sure what that means, you mean something like being able to visit the source's website on WebView?

**Comment by sofari23** [comment]
Yeah being able to visit the source WebView without having to install it
like in mihon

**Comment by null2264** [comment]
I see, I'll add it to the backlog for now

**Comment by sofari23** [comment]
Alright
I appreciate the swift response

**Comment by NoFace33** [comment]
To help, here's mihon commit [f776c36](https://github.com/mihonapp/mihon/commit/f776c36e7049eb04c6062c5aecbde10de52eedab)
